example/integration_test/t06_map_test.dart

```diff
@@ -27,280 +27,338 @@ import 'shared.dart';
 
 void main() {
   patrol('Test map types', (PatrolIntegrationTester $) async {
-    /// Set up navigation.
-    final GoogleNavigationViewController viewController =
-        await startNavigationWithoutDestination($);
-
-    // Test default type.
-    expect(await viewController.getMapType(), MapType.normal);
-
-    final List<MapType> types = <MapType>[
-      MapType.satellite,
-      MapType.terrain,
-      MapType.hybrid,
-      MapType.normal,
-    ];
-
-    for (final MapType type in types) {
-      await viewController.setMapType(mapType: type);
-      expect(await viewController.getMapType(), type);
+    for (final TestMapType testMapType in testMapTypes) {
```

Comment:
I would move the for loop outside of the patrol call.
And have the test type written in test name.

```dart
for (final TestMapType testMapType in testMapTypes) {
     patrol('Test map types (${testMapType.name})', (PatrolIntegrationTester $) async {
     ...
     });
     
     patrol('NEXT TEST IN SAME LOOP (${testMapType.name})', (PatrolIntegrationTester $) async {
     ...
     });
}
```